content/2020/11-17-client-credentials-grant.md

```diff
@@ -0,0 +1,40 @@
+---
+applied_at: "2020-11-17"
+applies_to: 
+- api
+is_impactful: false
+is_new_feature: false
+collapse: false
+show_excerpt: true
+release_source_url: ''
+---
+
+# Client Credentials Grant
+
+<!-- more -->
+
+Announcing a new way to leverage server to server, JWT, authentication:
+Client Credentials Grant. Prior to today, we required a public/private key pair
+and assertion to verify an application's identity and retrieve an Access Token.
```

Comment:
A few comments here. Firstly, the first sentence is missing a subject. Secondly, this new feature doesn't really have anything to do with JWT. It's an alternative to JWT really.

A good template for these changelog entries is to first give a 1 line description of what is new (We are introducing a new authenticated method called Client Credentials Grant), then explain what the impact is to the user (Nothing changes for existing apps, but for new apps this offers a far easier way to authenticate), and then finally go into full detail of the change (Prior to today.... etc)